components/org.wso2.bfsi.identity.extensions/src/main/java/org/wso2/bfsi/identity/extensions/auth/extensions/request/validator/BFSIRequestObjectValidationExtension.java

```diff
@@ -115,10 +115,11 @@ boolean validateIAMConstraints(RequestObject requestObject,
      */
     @Generated(message = "Empty method")
     @Override
-    protected boolean isValidAudience(RequestObject requestObject, OAuth2Parameters oAuth2Parameters) {
+    protected boolean isValidAudience(RequestObject requestObject, OAuth2Parameters oAuth2Parameters)
+            throws RequestObjectException {
 
         // converted to validation layer
-        return true;
+        return super.isValidAudience(requestObject, oAuth2Parameters);
```

Comment:
We don't need to override this method

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/12/commits/661110fc425c3513e35e90a631382085ca95d3ef